data/jegrami.json

```
@@ -0,0 +1,29 @@
+ {
```

You're missing some required properties link `name`, `bio`, etc. Please read the [QuickStart Docs](https://linkfree.io/docs/quickstart) for more information on how to add Your Profile. 

Please add your name, and bio as suggested below.

```suggestion
 {
  "name": "Jeremiah Igrami",
  "bio": "YOUR__BIO",
  "links": [
    {
```